pkg/cluster/kubefedcluster.go

```diff
@@ -0,0 +1,28 @@
+package cluster
+
+import (
+	"context"
+	errs "github.com/pkg/errors"
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// EnsureKubeFedClusterCrd creates a KubeFedCluster CRD in the cluster.
+// If the creation returns an error that is of the type "AlreadyExists" then the error is ignored,
+// if the error is of another type then it is returned
+func EnsureKubeFedClusterCrd(scheme *runtime.Scheme, client client.Client) error {
```

Comment:
I know it's a minor thing, but I believe that this func should be named `EnsureKubeFedClusterCRD` instead, since `CRD` is an acronym.

Reply:
Done.